### ISSUE TITLE
resoultion in makeTuneControlGrid

### ISSUE THREAD

**Issue by bommert** [mlr-org/mlr#1921]
In the help for makeTuneControlGrid it says for resolution:
"Either pass one resolution for all parameters, or a named vector."

But if I pass a named vector 
makeTuneControlGrid(resolution = c(x1 = 4, x2 = 5))

I get the error
Error in makeTuneControlGrid(resolution = c(x1 = 4, x2 = 5)) : 
  Assertion on 'resolution' failed: Must have length 1.

**Comment by pat-s** [comment]
```r
ps <- makeParamSet(makeDiscreteParam("x1", values = 4),
                   makeDiscreteParam("x2", values = 5))

ctrl <- makeTuneControlGrid()
```

**Comment by berndbischl** [comment]
> ps <- makeParamSet(makeDiscreteParam("x1", values = 4),
>                    makeDiscreteParam("x2", values = 5))
> 
> ctrl <- makeTuneControlGrid()

that looks incorrect. "values" should be vectors of values not scalars. 
but with makeDiscreteteParam you can easily discretize yourself.

WRT the OP:
Yes, thanks that looks like a little bug in the arg check

**Comment by berndbischl** [comment]
fixed in #1922 thx to @mb706
